digester/main.go

```diff
@@ -41,4 +147,113 @@ func main() {
 	err = db.Ping()
 	CheckError(err)
 	log.Println("Digester is now successfully connected to the tummy!")
+
+	err = unzipSource(ZIPFILE_PATH, EXTRACTION_DIR)
+	CheckError(err)
+	log.Println("Slack export has been successfully extracted!")
+
+	usersFile, err := os.ReadFile(EXTRACTION_DIR + string(os.PathSeparator) + USERS_FILENAME)
+	CheckError(err)
+	var users []User
+	err = json.Unmarshal(usersFile, &users)
+	CheckError(err)
+	log.Println("Digester found " + fmt.Sprint(len(users)) + " users.")
+
+	// Add slackbot as a user
+	slackbot := User{
```

Comment:
Can this be defined at the top-level `const`?

Reply:
structs cannot be constants in go

Reply:
go is great.